synapse/cores/common.py

```diff
@@ -2811,32 +2673,27 @@ def splicepump():
 
     def getCoreXact(self, size=1000):
         '''
-        Get a cortex transaction context for use in a with block.
-        This object allows bulk storage layer optimization and
-        proper ordering of events.
-
-        Example:
+        Get a Storage transaction context for use in a with block.
 
-            with core.getCoreXact() as xact:
-                core.dostuff()
+        This object allows bulk storage layer optimization and proper ordering
+        of events.  The context manager created through this function supports
+        firing splice events.
```

Comment:
While you're in here, can you add a note that `getCoreXact` may not be called over telepath?